presto-main/src/main/java/com/facebook/presto/util/Predicates.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.facebook.presto.sql.planner.optimizations;
+package com.facebook.presto.util;
```

Comment:
Also the class should be renamed to `MorePredicates` to be distinguished from Guava `Predicates` class. 
Also please notice #8325 if you run into conflicts :)